src/storm/templates/configmap.yaml

```diff
@@ -14,11 +14,13 @@ data:
 {{ include "storm.zookeeper.serverlist.yaml" $ | indent 6 }}
     storm.zookeeper.port: {{ include "storm.zookeeper.port" $ }}
     nimbus.seeds:
-      - {{ include "storm.nimbus.fullname" $ }}
+    {{- range $i, $e := until (int $.Values.nimbus.replicaCount) }}
+    - {{ template "storm.nimbus.fullname" $ }}-{{ $i }}.{{ template "storm.nimbus.fullname" $ }}
```

Comment:
I'm not totally sure if not indenting here is invalid yaml, but it would be nice to add two spaces so that it matches the `storm.zookeepers.servers` indentation above.